Задание1

ZoyaMi opened the pull request:
В объекте поменять местами имена и значения свойств

ZoyaMi committed:
задание1

В объекте поменять местами имена и значения свойств

ZoyaMi committed:
Добавлена ссылка на 1-й js

SergeBondar commented on 1js.js:
```diff
@@ -0,0 +1,13 @@
+var dev= {
```
отступы

SergeBondar commented on 1js.js:
```diff
@@ -0,0 +1,13 @@
+for (i in dev){
+var tmp=i;
+var tmp2=dev[i];
+delete dev[i];
```
операции удаления сложные
обычно создается новый объект, и в него все добавляется

SergeBondar commented on 1js.js:
```diff
@@ -0,0 +1,13 @@
+dev[tmp2]=tmp;
+}
+for (i in dev){
+console.log('property: ' + i + ' | ' + 'value: ' + dev[i]);
```
можно вывести сразу весь объект, так даже нагляднее будет

ZoyaMi committed:
Исправление замечаний

1. Добавлены отступы
2. Выполняется c помощью нового объекта 
3. Вывод всего объекта в консоль

SergeBondar commented on index.html:
```diff
@@ -7,11 +7,12 @@
     <h1>My home page</h1> 
     <h2>Who am I?</h2>
+    <script src="1js.js"></script>
   <p>Hi! Welcome to my Home Page! My name is Mr.Cat, I'm awesome and I know a little bit of HTML.</p>
```
а на каком уровне этот paragraph находится?

SergeBondar commented on 1js.js:
```diff
@@ -0,0 +1,9 @@
+    cool: 'yes',
+    stylish: 'no'
+}
+tmp = {};
```
объявление переменной?

SergeBondar commented on 1js.js:
```diff
@@ -0,0 +1,9 @@
+    stylish: 'no'
+}
+tmp = {};
+for (i in dev) {
```
объявление переменной?

ZoyaMi committed:
Добавление объявления переменной

tmp = {}; исправлено на var tmp = {};

ZoyaMi committed:
Исправлено оформление

Исправлено оформление - выровнен параграф

SergeBondar commented on 1js.js:
```diff
@@ -0,0 +1,9 @@
+var dev = {
+    cool: 'yes',
+    stylish: 'no'
+}
```
; после объявления переменной

SergeBondar commented on 1js.js:
```diff
@@ -0,0 +1,9 @@
+    stylish: 'no'
+}
+var tmp = {};
+for (i in dev) {
```
переменную i тоже нужно объявить for(var i in dev), тогда она будет жить столько, сколько нужно

ZoyaMi committed:
Добавлены var и ;

1. Добавлена ; после объявления переменной
2. Добавлено var в for (var i in dev)

ZoyaMi committed:
Ссылка на скрипт перемещена в конец боди
